src/tests/react-testing-library.test.tsx

```diff
@@ -1,34 +1,114 @@
 import React from 'react';
+import { fireEvent, logRoles } from '@testing-library/dom';
 import { render, screen } from '@testing-library/react';
 import { App } from '../components/app';
+import { Toggle, ToggleProps } from '../components/toggle/toggle';
+import { Button, ButtonProps } from '../components/button/button';
 
 describe('React Testing Library do:', () => {
 
     test('create snapshot of mounted component', () => {
-        throw new Error('Not migrated yet');
+        // render app
+        render(<App />);
+
+        // check the snapshot
+        expect(screen).toMatchSnapshot;
```

Comment:
`toMatchSnapshot` is called, but I can't find the new snapshot file. Didn't forget to commit something?